cockroachcloud/free-faqs.md

```diff
@@ -0,0 +1,92 @@
+
+## Security
+
+### Is my cluster secure?
+
+Yes. We create individual sub-accounts and VPCs for each cluster within the cloud provider. These VPCs are firewalled from each other and any other outside connection, unless allowlisted for SQL and Web UI ports.
+
+The allowlist is comprised of IP addresses that you provide to us, and is an additional layer of protection for your cluster. Connections will only be accepted if they come from an allowlisted IP address, which protects against both compromised passwords and any potential bugs in the server.
+
+We use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.2.
+
+### Is encryption-at-rest enabled on CockroachCloud Free?
+
+Yes. All data on CockroachCloud is encrypted-at-rest using the tools provided by the cloud provider that your cluster is running in.
+
+- Data stored in clusters running in GCP are encrypted-at-rest using [persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption).
+- Data stored in clusters running in AWS are encrypted-at-rest using [EBS encryption-at-rest](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
+
+Because we are relying on the cloud provider's encryption implementation (as noted above), we do not enable CockroachDB's [internal implementation of encryption-at-rest](../v20.2/encryption.html#encryption-at-rest-enterprise). This means that encryption will appear to be disabled in the [DB Console](../stable/ui-overview.html), since it is unaware of cloud provider encryption.
+
+### Is my cluster isolated? Does it share resources with any other clusters?
+
+CockroachCloud Free is a multi-tenant offering and resources are shared between clusters.
+
+## Cluster maintenance
+
+### How do I add nodes?
+
+We have eliminated the concept of nodes for CockroachCloud Free. However, there is an upper limit of usage of up to 1 vCPU and 5GB storage per free cluster.
+
+### Who is responsible for backups? Can I backup my cluster?
+
+Currently, backups are not supported on CockroachCloud Free. The ability to enable backups will be supported in the future.
+
+### Can I upgrade my cluster from CockroachCloud Free to the paid version of CockroachCloud?
+
+At this time, a CockroachCloud Free cluster cannot be upgraded. In the future, you will have the ability to move from CockroachCloud Free to a paid version of CockroachCloud.
+
+## Product features
+
+### Are enterprise features like partitioning or change data capture available to me?
+
+No, enterprise features are not available on CockroachCloud Free clusters. However, all enterprise features are available on the paid version of [CockroachCloud](create-your-cluster.html).
```

Comment:
I would add that at this moment, CDC and partitioning are not available but are coming soon. EAR is an enterprise feature and its available on CC Free so its not accurate to say that no enterprise features are available on CC Free.